.github/workflows/community-review.sh

```diff
@@ -208,34 +208,34 @@ process_pr_reviews() {
 
       #see if the user has read role
 
-      if [ "$push_permission" = 'true' ]; then
-          if [ "$state" = "APPROVED" ]; then
+      if [[ "$push_permission" == "true" ]]; then
+          if [[ "$state" == "APPROVED" ]]; then
              ((++committerApproves))
           fi
      else
           ((++communityReviews))
-          if [ "$state" = "APPROVED" ]; then
+          if [[ "$state" == "APPROVED" ]]; then
              ((++communityApproves))
           fi
      fi
 
-     if [ "$state" = "CHANGES_REQUESTED" ]; then
+     if [[ "$state" == "CHANGES_REQUESTED" ]]; then
         ((++requestForChanges))
      fi
   done <<< "$pr_reviews"
   echo "communityApproves $communityApproves requestForChanges $requestForChanges committerApproves $committerApproves communityReviews $communityReviews"
 
   local label_to_post=
   local label_to_delete=
-  if [[ $communityApproves -ge  "2" && $requestForChanges = 0 && $committerApproves = 0 ]]; then
+  if [[ $communityApproves -ge  2 && $requestForChanges = 0 && $committerApproves = 0 ]]; then
```

Comment:
Let's update all 3 parts of the condition to use arithmetic operators:
```bash
[[ $communityApproves -ge 2 && $requestForChanges -eq 0 && $committerApproves -eq 0 ]]
```

Also there is an extra whitespace at `-ge  2` now.

Reply:
@davidradl the `-eq` is still not applied to the 2nd and 3rd sub-conditions.

Reply:
yes - sorry I missed that -fixed